inferentia/README.md

```diff
@@ -122,7 +131,42 @@ loaded on cores 2-3. To best engage inferentia device, try setting
 the number of neuron cores to be a proper multiple of the instance
 count.
 
-The invocation should create a triton model directory with following
+### TensorFlow
+For TensorFlow, the model must be compiled for AWS Neuron. See
+[AWS Neuron TensorFlow](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/neuron-frameworks/tensorflow-neuron/tutorials/index.html
+tutorials to learn how to get a compiled model that uses Neuron
+cores. Currently, the code is tested only on `tensorflow==1.15`.
+
+Once the compiled model is obtained use [gen_triton_model.py](https://github.com/triton-inference-server/python_backend/blob/main/inferentia/scripts/gen_triton_model.py)

```

Comment:
Why are you using the entire url? Why not just use (scripts/gen_triton_model.py)? That way it reroutes to the specific branch instead of always pointing to main